packages/e2e-tests/config/setup-test-framework.js

```diff
@@ -68,6 +68,11 @@ const pageEvents = [];
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout( PUPPETEER_TIMEOUT || 100000 );
 
+// Retry flaky tests at most 2 times in CI.
```

Comment:
Does this just retry flaky tests, or all failing tests?

Reply:
Oh okay so this is the foundation of how we detect flaky tests in the first place. Should probably rephrase it to "retry failing tests".

- If a test fails, it will automatically re-run at most 2 times.
- If all 3 times fail, it's then reported as a **failing** test. Not a flaky test and won't report it to issue.
- If it passes after retrying, it's then marked as a **flaky** test, and will post to issue, but displayed as passed in the original commit.